parts/k8s/kubernetesparams.t

```diff
@@ -246,7 +246,8 @@
          "1.3.7",
          "1.3.8",
          "1.3.9",
-         "1.4.4"
+         "1.4.4",
+         "1.4.6"
```

Comment:
You have a list of valid versions - is there a reason for this?  Does this list translate to versions already on the VHD?

Reply:
It's largely there for historical purposes, it's the set of "known-working" (i.e., tested with Kubernetes on Azure in the aks-engine CSE) versions. And because there is a facility to accept an arbitrary containerd version string in the api model.